packages/core/src/__tests__/cogniteClient.unit.spec.ts

```diff
@@ -131,6 +131,70 @@ describe('CogniteClient', () => {
 
         expect(getToken).toHaveBeenCalledTimes(1);
       });
+
+      test('getToken should be called once for parallel 401s', async () => {
+        nock(mockBaseUrl).get('/test').thrice().reply(401);
+        nock(mockBaseUrl).get('/test').thrice().reply(200);
+
+        const mockGetToken = jest.fn(
+          () =>
+            new Promise<string>((resolve) => {
+              setTimeout(() => {
```

Comment:
Just a heads up: https://jestjs.io/docs/timer-mocks

Reply:
I didn't know this existed, so TIL thanks 🙏🏻

Reply:
Lowered it to 100ms

Reply:
Yes, that's fine. At least you are now aware of the feature 🙂